index.html

```diff
@@ -55,6 +55,7 @@
   <footer>KMA's Software Development Portfolio | &copy; 2016</footer>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.min.js"></script>
+  <script src="projectInfo.json"></script>
```

Comment:
You should also remove your projectInfo.js file completely from your project (and remove the script tag).